numpy/core/arrayprint.py

```diff
@@ -1537,16 +1546,16 @@ def array_str(a, max_line_width=None, precision=None, suppress_small=None):
     a : ndarray
         Input array.
     max_line_width : int, optional
-        Inserts newlines if text is longer than `max_line_width`.  The
-        default is, indirectly, 75.
+        Inserts newlines if text is longer than `max_line_width`.
+        See `linewidth` in `set_printoptions` for defaults.
```

Comment:
How about documenting this as ```defaults to ``np.get_printoptions()['linewidth']`` ``` (note: double backticks for RST code)

Reply:
Ah I missed that. Yes that makes more sense. I'll update.